strawberryfield.routing.yml

```diff
@@ -26,4 +26,16 @@ strawberryfield.important_solr_settings_form:
   requirements:
     _permission: 'access administration pages'
   options:
-    _admin_route: TRUE
\ No newline at end of file
+    _admin_route: TRUE
+    
+strawberryfield.custom_node_edit: 
+  path: '/node/{node}/custom-edit'
```

Comment:
Since this is the actual routing. Do we need to have /node here? If we are normalizing things as /do? Not relevant really, just trying to understand the argument against/favor of going this route. Also, please change `custom-edit` to something that feels less exposing internals to end users. Remember that the most obvious use case of this is a user with very limited credentials, so having a path on the browser that says custom-edit will make them wonder what a standard-edit looks like and why they don't have that.

Reply:
@DiegoPino Good point I will change the name.
I with that path, I was basically trying to copy the form of the standard routes, which are `/node/<nid>/edit` , `/node/<nid>/delete`, etc. But there is nothing mandating this.